apps/docs/pages/guides/cli/using-envitonment-variables-in-your-config.mdx

```diff
@@ -0,0 +1,34 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'using-envitonment-variables-in-your-config',
+  title: 'Using environment variables in your config.toml',
+  description: 'How to use environment variables in your config.toml with the Supabase CLI.'
+}
+
+The Supabase CLI can use environment variables from your `.env` file in your project's root directory inside 
+your `config.toml` file.
```

Comment:
```suggestion
The Supabase CLI can use environment variables from your `.env` file in your project's root directory inside your `config.toml` file.
```

Could you please disable line-breaks for markdown files? It makes it harder to make suggestions in PRs

VSCode (and other editors) usually have a "wrap" configuration item which helps with readability on your local machine

Reply:
Will do.